docs/help.md

```diff
@@ -12,16 +12,13 @@
   - *title* - the title of the issue
   - *body* - the contents of the issue
 
-`bot links` - Return gdocs url for tag `+link`
+`bot links` - Return google spreadsheet url for tag `+link`
```

Comment:
think Google SpreadSheet (Camelcased) looks better

Reply:
will be fixed